pilot-app/src/app.vue

```diff
@@ -0,0 +1,156 @@
+﻿<template>
+    <article class="bodySection" v-if="isLoggedIn">
+        <!-- <status-bar :message="statusMessage" @ondestroy="statusMessage = undefined"></status-bar> -->
+        <section class="bodyHeader">
+            <top-navigation
+                v-if="content && (content.topNavigation || content.sideNavigation)"
+                :content="content.topNavigation"
+                :profile="content.profile"
+                :channel="content.channel"
+                :isMobile="isMobile"
+                :sideNavigationContent="content.sideNavigation"
+                @onchange="onTopNavChange"
+            ></top-navigation>
+        </section>
+        <section class="bodyArticle">
+            <aside class="bodyAside">
+                <sideNavigation v-if="!isMobile && content && content.sideNavigation" :content="content.sideNavigation"></sideNavigation>
+            </aside>
+            <section class="bodyContent">
+                <listView ref="list" :IDs="pageIDs"></listView>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 16 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
            </section>
```